pkg/handlers/handlers.go

```diff
@@ -339,14 +339,9 @@ func RegisterUnauthenticatedRoutes(handler *Handler, kotsStore store.Store, debu
 	loggingRouter.Path("/api/v1/troubleshoot/supportbundle/{bundleId}/redactions").Methods("PUT").HandlerFunc(handler.SetSupportBundleRedactions)
 	loggingRouter.Path("/api/v1/preflight/app/{appSlug}/sequence/{sequence}").Methods("POST").HandlerFunc(handler.PostPreflightStatus)
 
-	// This the handler for license API and should be called by the application only.
+	// This these handlers for should be called by the application only.
```

Comment:
nit pick on the comment
```suggestion
	// These handlers should be called by the application only.
```